terraform/main.tf

```diff
@@ -245,4 +245,26 @@ module "webapp-logs-s3" {
   application_name    = "webapp"
 }
 
-
+module "route53" {
+  source                                       = "./modules/route53"
+  root_domain_name                             = var.root_domain_name
+  create_hosted_zone                           = var.create_hosted_zone
+  production_webapp_alb_dns                    = var.production_webapp_alb_dns
+  production_backoffice_alb_dns                = var.production_backoffice_alb_dns
+  staging_webapp_alb_dns                       = var.staging_webapp_alb_dns
+  staging_backoffice_alb_dns                   = var.staging_backoffice_alb_dns
+  dev_webapp_alb_dns                           = var.dev_webapp_alb_dns
+  dev_backoffice_alb_dns                       = var.dev_backoffice_alb_dns
+  dev_backoffice_ssl_verification_name         = var.dev_backoffice_ssl_verification_name
+  dev_backoffice_ssl_verification_value        = var.dev_backoffice_ssl_verification_value
+  dev_webapp_ssl_verification_name             = var.dev_webapp_ssl_verification_name
+  dev_webapp_ssl_verification_value            = var.dev_webapp_ssl_verification_value
+  production_backoffice_ssl_verification_name  = var.production_backoffice_ssl_verification_name
+  production_backoffice_ssl_verification_value = var.production_backoffice_ssl_verification_value
+  production_webapp_ssl_verification_name      = var.production_webapp_ssl_verification_name
+  production_webapp_ssl_verification_value     = var.production_webapp_ssl_verification_value
+  staging_backoffice_ssl_verification_name     = var.staging_backoffice_ssl_verification_name
+  staging_backoffice_ssl_verification_value    = var.staging_backoffice_ssl_verification_value
+  staging_webapp_ssl_verification_name         = var.staging_webapp_ssl_verification_name
+  staging_webapp_ssl_verification_value        = var.staging_webapp_ssl_verification_value
```

Comment:
Can we pass in a list of maps here perhaps from a tf-vars file
eg
```envs = [{
  webapp_alb_dns
  backoffice_alb_dns  
  backoffice_ssl_verification_name
  backoffice_ssl_verification_value
  webapp_ssl_verification_name 
  webapp_ssl_verification_value
}```

Reply:
I'd be happy to do it this way if we're sure the DNS validation URLs aren't sensitive info.

Reply:
I wonder if we can generate a wildcard cert for the whole hosted zone.  ie *.ourhostedzone.com., ourhostedzone.com

Reply:
If the  webapp_ssl_verification_value is the ACME challenge we definitely shouldn't be passing it in like I've suggested. If that's the case maybe we could use ACM in prod for all the accounts?